system/Session/Handlers/DatabaseHandler.php

```diff
@@ -317,7 +317,7 @@ public function destroy($sessionID): bool
      */
     public function gc($maxlifetime): bool
     {
-        $interval = implode(" '"[(int)($this->platform === 'postgre')], ['', "{$maxlifetime} second", '']);
+        $interval = implode(" '"[(int) ($this->platform === 'postgre')], ['', "{$maxlifetime} second", '']);
```

Comment:
The cast is approved, but... what is this mess? Holy cow that's hard to read.

Reply:
When I'm manually checking the changes I also did not understand this witchcraft. 😂

Reply:
Apparently, this is perfectly legal in all PHP versions. https://3v4l.org/Bkf7k
That's news for me.

Reply:
Ah I get it. It is array access on a string. 😅

Reply:
I reworked this to get explicitly the `$glue` first.